content/en/service_management/app_builder/embedded_apps.md

```diff
@@ -93,10 +93,25 @@ To add a button that sets the value of a date range picker component to the dash
     - **Value**: `${global?.dashboard?.timeframe}`
 1. Save and publish your app.
 
+## Add apps to Software Catalog
+
+Add a published app to the [Self-Service][2] section of [Software Catalog][3] to provision infrastructure, scaffold services, remediate issues, and more, all from a central place.
+
+To add to Self-Service, first ensure your app is published and permissions are defined. Next you can click `Add to Self-Service`.
```

Comment:
```suggestion
To add to Self-Service, first ensure your app is published and permissions are defined. Next, you can click **Add to Self-Service**.
```

Where does **Add to Self-Service** appear? Usually we describe which page or section the text is in, so the user knows where to look for it.